slsDetectorCalibration/jungfrauExecutables/Makefile.rawdataprocess

```diff
@@ -2,10 +2,11 @@
 # Copyright (C) 2021 Contributors to the SLS Detector Package
 #module add CBFlib/0.9.5
 
+INCDIR=-I. -I../ -I../interpolations  -I../interpolations/etaVEL  -I../dataStructures  -I../../slsSupportLib/include/ -I../../slsReceiverSoftware/include/ -I../tiffio/include -I/afs/psi/project/sls_det_software/conda/envs/pyclang_fmt/include/
 
-INCDIR=-I. -I../ -I../interpolations  -I../interpolations/etaVEL  -I../dataStructures  -I../../slsSupportLib/include/ -I../../slsReceiverSoftware/include/ -I../tiffio/include
-
-LDFLAG= ../tiffio/src/tiffIO.cpp -L/usr/lib64/ -lpthread -lm  -lstdc++   -pthread -lrt    -ltiff -O3  -std=c++11
+LDFLAG= ../tiffio/src/tiffIO.cpp  -L/afs/psi/project/sls_det_software/conda/envs/pyclang_fmt/lib/ -L/usr/lib64/ -lpthread -lm  -lstdc++   -pthread -lrt    -ltiff -O3  -std=c++11 -lfmt
```

Comment:
This is a good advertisement for CMake since we can avoid having to hardcode paths. I'll still merge this, but hopefully we can delete the makefile soon.